spec/faraday/adapter/excon_spec.rb

```diff
@@ -1,5 +1,5 @@
 RSpec.describe Faraday::Adapter::Excon do
-  features :body_on_get, :reason_phrase_parse
+  features :body_on_get, :reason_phrase_parse, :trace_method, :connect_method
```

Comment:
Is this because not all adapters support these verbs?

Reply:
Yup, Patron :)

```
  51) Faraday::Adapter::Patron behaves like an adapter with SSL disabled #connect behaves like a request method handles headers with multiple values
      Got 1 failure and 1 other error:
      Shared Example Group: "a request method" called from ./spec/support/shared_examples/adapter.rb:56
      Shared Example Group: "adapter examples" called from ./spec/support/shared_examples/adapter.rb:11
      Shared Example Group: "an adapter" called from ./spec/faraday/adapter/patron_spec.rb:4

      51.1) Failure/Error: session.request(env[:method], env[:url].to_s, env[:request_headers], :data => data)

            ArgumentError:
              Action must be one of GET, PUT, POST, DELETE, HEAD, COPY, PATCH, OPTIONS
            # ./lib/faraday/adapter/patron.rb:30:in `call'
            # ./lib/faraday/response.rb:8:in `call'
            # ./lib/faraday/request/url_encoded.rb:20:in `call'
            # ./lib/faraday/request/multipart.rb:19:in `call'
            # ./lib/faraday/rack_builder.rb:151:in `build_response'
            # ./lib/faraday/connection.rb:485:in `run_request'
            # ./lib/faraday/connection.rb:196:in `connect'
            # ./spec/support/shared_examples/request_method.rb:3:in `public_send'
            # ./spec/support/shared_examples/request_method.rb:3:in `block (2 levels) in <top (required)>'
            # ./spec/support/shared_examples/request_method.rb:13:in `block (2 levels) in <top (required)>'
```

It may be easy to fix upstream, but then Faraday would have to depend on a more specific version of Patron.

Reply:
Ah, shame on Patron! Fair enough, thanks for confirming :)